src/lib.rs

```diff
@@ -0,0 +1,55 @@
+// We implement PackageVersion ourselves to avoid creating a WinRT dependency.
+
+#[repr(C)]
+#[derive(Clone, Copy)]
+struct PackageVersion {
+    major: u8,
+    minor: u8,
+    build: u8,
+    revision: u8,
+}
```

Comment:
Fix `PACKAGE_VERSION` ABI.

```suggestion
struct PackageVersion {
    revision: u8,
    build: u8,
    minor: u8,
    major: u8,
}
```